core/storage/question/gae_models.py

```diff
@@ -17,7 +17,7 @@
 from core.platform import models
 import utils
 
-from google.appengine.ext import ndb
+from google.appengine.ext import ndb  # pylint: disable=relative-import
```

Comment:
Hi @seanlip! @aks681 had discussed this with me. For relative-import errors, we disable the check for the corresponding line similar to [these lines](https://github.com/oppia/oppia/blob/develop/constants_test.py#L19:L20) in `./constants_test.py`.  
But I tested it locally just now and I do not face any error or problem without the disable. Also, the develop branch does not have the disable statement and all checks pass. Let me talk to @aks681 once again regarding this. Thanks!

Reply:
Yeah, in travis they pass, but when running pre_submit checks in my local machine, they fail. Even, when I push develop also, this happens.

Reply:
@aks681: All tests pass locally on my machine. Similarly for other developers. Also, this check has not been recently added, it was already enabled in Pylint. So I think it might be a local machine problem, not sure though. Thanks!

Reply:
Ok, removed it.

Reply:
@aks681 -- for reference, how did you resolve this?